org-formation/720-client-vpn/vpn.njk

```diff
@@ -133,7 +145,7 @@ Resources:
     Properties:
       Description: {{ spoke }}-Subnet{{ subnet_id | last }}
       ClientVpnEndpointId: !Ref Endpoint
-      DestinationCidrBlock: {{ spoke_data.CIDR }}
+      DestinationCidrBlock: 0.0.0.0/0
```

Comment:
Probably shouldn't do this, actually... but instead leave the individual cidr ranges to spokes in the templates. If so, there needs to an explicit route to `0.0.0.0/0` to catch any traffic destined outside of the private ranges